adam-core/src/main/scala/org/bdgenomics/adam/converters/FastqRecordConverter.scala

```diff
@@ -41,6 +41,96 @@ import scala.collection.JavaConversions._
 private[adam] class FastqRecordConverter extends Serializable with Logging {
 
   /**
+   * Parse 4 lines at a time
+   * @see parseReadPairInFastq
+   * *
+   */
+  private def parseReadInFastq(input: String,
```

Comment:
Is the use of `private` like this a good practice? It will be used in other methods in `FastqRecordConverter` via `this.parseReadInFastq`